Test failure from installation page

## Expected Behavior
Tests should complete. Tests are run using command 
`py.test --ignore cwltool/schemas/ --pyarg cwltool`

## Actual Behavior
Test failed. Output:

`==================================================== FAILURES ====================================================
__________________________________________ TestOverride.test_overrides ___________________________________________

self = <cwltool.tests.test_override.TestOverride testMethod=test_overrides>

    @pytest.mark.skipif(onWindows(),
                        reason="Instance of Cwltool is used, On windows that invoke a default docker Container")
    def test_overrides(self):
        sio = StringIO()
    
        self.assertEquals(main([get_data('tests/override/echo.cwl'),
                                get_data('tests/override/echo-job.yml')],
                               stdout=sio), 0)
>       self.assertEquals({"out": "zing hello1"}, json.loads(sio.getvalue()))
E       AssertionError: {'out': 'zing hello1'} != {u'out': u'-n zing hello1\n'}
E       - {'out': 'zing hello1'}
E       + {u'out': u'-n zing hello1\n'}
E       ?  +       + +++           ++

anaconda/envs/CWLtest/lib/python2.7/site-packages/cwltool/tests/test_override.py:26: AssertionError`

## Workflow Code
```
`py.test --ignore cwltool/schemas/ --pyarg cwltool`

```

## Full Traceback
```pytb
Paste the full traceback in case there is an exception
Run the workflow with ``--debug`` flag for more verbose logging 
```

## Your Environment
* cwltool version: 
`cwltool 1.0.20180116213856`
Check using ``cwltool --version``
OS: Mac OS 10.12.6

--- a/tests/override/echo.cwl
+++ b/tests/override/echo.cwl
@@ -15,5 +15,5 @@ outputs:
       glob: out.txt
       loadContents: true
       outputEval: $(self[0].contents)
-arguments: ["echo", "-n", $(inputs.m1), {shellQuote: false, valueFrom: "$MESSAGE"}]
+arguments: ["echo", $(inputs.m1), {shellQuote: false, valueFrom: "$MESSAGE"}]
 stdout: out.txt

--- a/tests/test_override.py
+++ b/tests/test_override.py
@@ -23,25 +23,25 @@ class TestOverride(unittest.TestCase):
         self.assertEquals(main([get_data('tests/override/echo.cwl'),
                                 get_data('tests/override/echo-job.yml')],
                                stdout=sio), 0)
-        self.assertEquals({"out": "zing hello1"}, json.loads(sio.getvalue()))
+        self.assertEquals({"out": "zing hello1\n"}, json.loads(sio.getvalue()))
 
         sio = StringIO()
         self.assertEquals(main(["--overrides", get_data('tests/override/ov.yml'),
                                 get_data('tests/override/echo.cwl'),
                                 get_data('tests/override/echo-job.yml')],
                                stdout=sio), 0)
-        self.assertEquals({"out": "zing hello2"}, json.loads(sio.getvalue()))
+        self.assertEquals({"out": "zing hello2\n"}, json.loads(sio.getvalue()))
 
         sio = StringIO()
         self.assertEquals(main([get_data('tests/override/echo.cwl'),
                                 get_data('tests/override/echo-job-ov.yml')],
                                stdout=sio), 0)
-        self.assertEquals({"out": "zing hello3"}, json.loads(sio.getvalue()))
+        self.assertEquals({"out": "zing hello3\n"}, json.loads(sio.getvalue()))
 
         sio = StringIO()
         self.assertEquals(main([get_data('tests/override/echo-job-ov2.yml')],
                                stdout=sio), 0)
-        self.assertEquals({"out": "zing hello4"}, json.loads(sio.getvalue()))
+        self.assertEquals({"out": "zing hello4\n"}, json.loads(sio.getvalue()))
 
 
         sio = StringIO()
@@ -49,18 +49,18 @@ class TestOverride(unittest.TestCase):
                                 get_data('tests/override/echo-wf.cwl'),
                                 get_data('tests/override/echo-job.yml')],
                                stdout=sio), 0)
-        self.assertEquals({"out": "zing hello2"}, json.loads(sio.getvalue()))
+        self.assertEquals({"out": "zing hello2\n"}, json.loads(sio.getvalue()))
 
         sio = StringIO()
         self.assertEquals(main(["--overrides", get_data('tests/override/ov2.yml'),
                                 get_data('tests/override/echo-wf.cwl'),
                                 get_data('tests/override/echo-job.yml')],
                                stdout=sio), 0)
-        self.assertEquals({"out": "zing hello5"}, json.loads(sio.getvalue()))
+        self.assertEquals({"out": "zing hello5\n"}, json.loads(sio.getvalue()))
 
         sio = StringIO()
         self.assertEquals(main(["--overrides", get_data('tests/override/ov3.yml'),
                                 get_data('tests/override/echo-wf.cwl'),
                                 get_data('tests/override/echo-job.yml')],
                                stdout=sio), 0)
-        self.assertEquals({"out": "zing hello6"}, json.loads(sio.getvalue()))
+        self.assertEquals({"out": "zing hello6\n"}, json.loads(sio.getvalue()))

